ansys/dpf/core/plotter.py

```diff
@@ -601,10 +601,7 @@ def plot_chart(fields_container, off_screen=False, screenshot=None):
     def plot_contour(
             self,
             field_or_fields_container,
-            notebook=None,
             shell_layers=None,
-            off_screen=None,
-            show_axes=True,
```

Comment:
@PProfizi Would it be possible to check that the show_axes (not used in any examples I think) is still working ?

Reply:
It is in the new example where all previous arguments are shown explicitly, with their default PyVista values.